dspace-api/src/main/java/org/dspace/content/service/ItemService.java

```diff
@@ -539,14 +539,15 @@ public void move(Context context, Item item, Collection from, Collection to, boo
         throws SQLException, AuthorizeException, IOException;
 
     /**
-     * Check the bundle ORIGINAL to see if there are any uploaded files
+     * Check the bundle ORIGINAL or METADATA to see if there are any uploaded files
```

Comment:
are you sure, cannot bundleName be anything?

Reply:
changed comment